config/locales/en/faker/offence.yml

```diff
@@ -803,15 +803,15 @@ en:
         - Summary offence
       mode_of_trial_reason:
         - Offence triable only by court martial
-        - Indictable-only offence
-        - No mode of Trial - Either way offence
+        - Indictable only offence
```

Comment:
Is there a reason these are changing? They come from here: https://docs.google.com/spreadsheets/d/1JD6vENSmPTcZIB_Lo0wtp6acXY2k4McN5WrUVbBpEgA/edit#gid=925794357

If it's so it matched `AllocationDecision::DESCRIPTIONS`, I'd prefer if we changed those to match these.

Reply:
Hyphens removed to match the underlying schema

Reply:
Which schema are you referring to here? I only know about these descriptions and codes coming from that google doc I shared the link for. If you have another source for what these values will be could you share it please? It would be useful to have.